parsec/core/gui/validators.py

```diff
@@ -38,9 +38,7 @@ def validate(self, string, pos):
             BackendAddr.from_url(string)
             return QValidator.Acceptable, string, pos
         except ValueError:
-            if pos > 3:
-                return QValidator.Invalid, string, pos
-            return QValidator.Intermediate, string, pos
+            return QValidator.Invalid, string, pos
```

Comment:
The remove of the `pos > 3 check` comes from changes in https://github.com/Scille/parsec-cloud/pull/1402 ?

Reply:
I think the `pos > 3` dated from way back when we didn't even have a `parsec://` scheme. No idea what was the point of it so I just removed it.